### PR TITLE
samples: Correct CAF sample documentation

### PR DESCRIPTION
**Pull request by emob-nordic**
This commit corrects the CAF sample README file missed during initial PR.

Signed-off-by: Emil Obalski <emil.obalski@nordicsemi.no>

### CODE REVIEW

File: samples/caf/README.rst [hunk 1]
```diff
@@ -50,9 +50,9 @@ User interface
...
   Changes the LED effect displayed on the LED to the next defined LED effect:
 
-  * The first default LED effect toggles the blinking of the LED, with the period set to 200 ms.
+  * The first default LED effect toggles the blinking of the LED, with the period set to 400 ms.
   * The second default LED effect toggles the breathing of the LED, with the period set to 400 ms.
-  * The third default LED effect toggles a looped effect where blinks with the period set to 200 ms and then turns off for 1000 ms.
+  * The third default LED effect toggles a looped effect where blinks with the period set to 400 ms and then turns off for 1000 ms.
 
 Button 2:
    Switches the active LED: turns off the currently active LED and turns on the next available LED.
```
Comment (wiba-nordic) [inline samples/caf/README.rst:58]: Does this line need to change as well?

Comment (emob-nordic) [reply samples/caf/README.rst:58]: Yes! I missed that, thanks for spotting...

Comment (emob-nordic) [reply samples/caf/README.rst:58]: @wiba-nordic  What do you think about:
```
Switches the active LED: leaves the currently active LED with displayed LED effect and turns on the next available LED.

Comment (wiba-nordic) [reply samples/caf/README.rst:58]: I'd maybe go with:
`Switches the active LED to the next available LED. 
The previously active LED keeps its LED effect, and the new active LED turns on.`

File: samples/caf/README.rst [hunk 2]
```diff
@@ -50,12 +50,12 @@ User interface
...
 
-  * The first default LED effect toggles the blinking of the LED, with the period set to 200 ms.
+  * The first default LED effect toggles the blinking of the LED, with the period set to 400 ms.
   * The second default LED effect toggles the breathing of the LED, with the period set to 400 ms.
-  * The third default LED effect toggles a looped effect where blinks with the period set to 200 ms and then turns off for 1000 ms.
+  * The third default LED effect toggles a looped effect where blinks with the period set to 400 ms and then turns off for 1000 ms.
 
 Button 2:
-   Switches the active LED: turns off the currently active LED and turns on the next available LED.
+   Switches the active LED: Switches the active LED to the next available LED. The previously active LED keeps its LED effect, and the new active LED turns on.
```
Comment (wiba-nordic) [inline samples/caf/README.rst:58]:
```suggestion
   Switches the active LED: Switches the active LED to the next available LED. 
   The previously active LED keeps its LED effect, and the new active LED turns on.
```

**Review by wiba-nordic — APPROVED** [review]
Looks good.

File: samples/caf/README.rst [hunk 3]
```diff
@@ -50,12 +50,13 @@ User interface
 Button 1:
   Changes the LED effect displayed on the LED to the next defined LED effect:
 
-  * The first default LED effect toggles the blinking of the LED, with the period set to 200 ms.
+  * The first default LED effect toggles the blinking of the LED, with the period set to 400 ms.
   * The second default LED effect toggles the breathing of the LED, with the period set to 400 ms.
-  * The third default LED effect toggles a looped effect where blinks with the period set to 200 ms and then turns off for 1000 ms.
+  * The third default LED effect toggles a looped effect where blinks with the period set to 400 ms and then turns off for 1000 ms.
```
Comment (MarekPieta) [inline samples/caf/README.rst:55]: `... where LED blinks three times with period of 400 ms ....`?

**Review by MarekPieta — APPROVED** [review]

File: samples/caf/README.rst (hunk 3 above)
Comment (emob-nordic) [reply samples/caf/README.rst]: fixed

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**